packages/contracts-bedrock/test/universal/OptimismMintableERC20Factory.t.sol

```diff
@@ -1,15 +1,65 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.15;
 
-import { OptimismMintableERC20 } from "src/universal/OptimismMintableERC20.sol";
+// Testing utilities
 import { Bridge_Initializer } from "test/setup/Bridge_Initializer.sol";
+import { NextImpl } from "test/mocks/NextImpl.sol";
+import { EIP1967Helper } from "test/mocks/EIP1967Helper.sol";
+
+// Target contract dependencies
+import { OptimismMintableERC20 } from "src/universal/OptimismMintableERC20.sol";
+import { Proxy } from "src/universal/Proxy.sol";
+
+// Target contract
+import { OptimismMintableERC20Factory } from "src/universal/OptimismMintableERC20Factory.sol";
 
 contract OptimismMintableTokenFactory_Test is Bridge_Initializer {
     event StandardL2TokenCreated(address indexed remoteToken, address indexed localToken);
     event OptimismMintableERC20Created(address indexed localToken, address indexed remoteToken, address deployer);
 
-    function test_bridge_succeeds() external {
-        assertEq(address(l2OptimismMintableERC20Factory.BRIDGE()), address(l2StandardBridge));
+    /// @dev Tests that the constructor is initialized correctly.
+    function test_constructor_succeeds() external {
+        OptimismMintableERC20Factory impl = new OptimismMintableERC20Factory();
+        assertEq(address(impl.BRIDGE()), address(0));
+        assertEq(address(impl.bridge()), address(0));
+    }
+
+    /// @dev Tests that the proxy is initialized correctly.
+    function test_initialize_succeeds() external {
+        assertEq(address(l1OptimismMintableERC20Factory.BRIDGE()), address(l1StandardBridge));
+        assertEq(address(l1OptimismMintableERC20Factory.bridge()), address(l1StandardBridge));
+    }
+
+    /// @dev Tests that the proxy cannot be initialized twice.
+    function test_initialize_cannotInitProxy_reverts() external {
```

Comment:
If you update `packages/contracts-bedrock/test/vendor/Initializable.t.sol` then we have test coverage of this and this particular test can be deleted